spaceship/spec/portal/persons_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe Spaceship::Portal::Persons do
+  before { Spaceship.login }
+  let(:client) { Spaceship::Portal::App.client }
+  it "Should load all persons" do
```

Comment:
Usually I don't think we have 2 separate files for `person` and `persons`, any reason why those are separate?